app/config/config_test.php

```diff
@@ -70,11 +70,11 @@
         'connections'        => [
             'default' => [
                 'driver'   => 'pdo_mysql',
-                'host'     => getenv('DB_HOST') ?: '%mautic.db_host%',
-                'port'     => getenv('DB_PORT') ?: '%mautic.db_port%',
-                'dbname'   => getenv('DB_NAME') ?: '%mautic.db_name%',
-                'user'     => getenv('DB_USER') ?: '%mautic.db_user%',
-                'password' => getenv('DB_PASSWD') ?: '%mautic.db_password%',
+                'host'     => '%env(DB_HOST)%' ?: '%mautic.db_host%',
```

Comment:
@escopecz we need always to inject `env` to containers like this otherwise it will break.

@RCheesley this is worth documenting.